### ISSUE TITLE
Adding new device doesn't work

### ISSUE THREAD

**Issue by victorfiji** [loki-project/session-ios#139]
**Describe the bug**
When adding my iPhone device to my existing account I keep getting "waiting for authentication.."

**To reproduce**
1. Make session account on windows
2. Use iOS app and try to link account (either through ID or QR, I tried both)

**Screenshots or logs**
Self explanatory

**Smartphone (please complete the following information):**
 - Device: iPhone 8
 - OS: 13.3.1
 - Version of Loki Messenger or latest commit hash: 1.0.6

**Additional context**

**Comment by victorbrowser** [comment]
I just tried for the third time and it worked (third times the charm?) however this still happened twice so I guess it's worth looking into.

**Comment by robohack** [comment]
I'm having absolutely no luck linking my iPhone to an account created on my OSX desktop instance of Session.

The phone quickly scans the QR code, but most of the time the registration request never even makes it back to the desktop (the QR code is displayed until cancelled).  On rare occasions when the linking request does get back to the desktop it has often taken several minutes to do so.  Then once the request is OKed on the desktop the reply never makes it back to the iPhone, not even after hours have gone by.  Meanwhile when it OKs a linking request the desktop instance says the device has been successfully linked.

I've tried dozens of times today with no success.

My local internet connection is fast and very stable, with the iPhone using that same connection via WiFi only.

Given the note in the blog post about how pairing was implemented, I suspect "the official Loki File Server" is overloaded and perhaps sometimes dropping or ignoring requests.  That would at least explain why pairing never gets past the first step sometimes.

As for why pairing never completes on the secondary device, well I can only guess there's also a bug somewhere exacerbating the problem.
